### PR TITLE
my HW7..

### PR DESCRIPTION
**Pull request by TroianSergii**
Please check my HW 7
thanks!

### CODE REVIEW

File: test/07/HW7.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        user_input2 = input('Please enter a name you want to delete: ')
+        del phone_book[user_input2]
+        print(f" Name {user_input2} was successfully deleted")
+    else:
+        print("""This command in not supported, please use one from the list:
+        stats - number of records
+        list - a list of all the names in the book
+        show - detailed information by name
+        add- to add a record <name>,<number>
+        delete <name> - to delete record by name """)
```
Comment (klymenok) [inline test/07/HW7.py:41]: This description shows the wrong way how to use your program. Please, change it according to the read implementation

File: test/07/HW7.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            name = split_input2[0]
+            p_number = split_input2[1]
+            if name in phone_book:
+                print(f""" the name '{name}' is already in the book,
+                 you can't change the number, to modify please delete it first""")
+            else:
+                phone_book[name] = p_number
+    elif user_input == 'delete': # delete <name>: видалити запис за іменем (ключем)
+        user_input2 = input('Please enter a name you want to delete: ')
+        del phone_book[user_input2]
```
Comment (klymenok) [inline test/07/HW7.py:33]: It is better to check if key exists before delete it, otherwise you will have an error for non-existing key

**Review by klymenok — CHANGES_REQUESTED** [review]

File: test/07/HW7.py (hunk 1 above)
Comment (TroianSergii) [reply test/07/HW7.py]: I corrected the comments, but I'm unsure if I understood your request.
Thanks!

File: test/07/HW7.py (hunk 2 above)
Comment (TroianSergii) [reply test/07/HW7.py]: Corrected

File: test/07/HW7.py (hunk 1 above)
Comment (klymenok) [reply test/07/HW7.py]: I mean, when it is written `delete <name>` I try to use this command like `delete abonent1`, but your program says that there is no such command. Fix the description of your commands. It this particular case, fix this line to `delete - to delete contact by its name """)`

File: test/07/HW7.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        if user_input2 not in phone_book:
+            print(f"The name '{user_input2}' is not in the book")
+        else:
+            del phone_book[user_input2]
+            print(f" Name {user_input2} was successfully deleted")
+    else:
+        print("""This command is not supported, please use one from the list:
+        stats - show number of contact in the book
+        list - full list of all contact in the book
+        show <name> - detailed information about contact by its name
```
Comment (klymenok) [inline test/07/HW7.py:42]: Change description here as well, because your program doesn't work when I put `show abonent1`

**Review by klymenok — CHANGES_REQUESTED** [review]

File: test/07/HW7.py (hunk 1 above)
Comment (TroianSergii) [reply test/07/HW7.py]: Noted. Made changes
Please check again.
Thanks!

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**